src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonPropertiesView.ts

```diff
@@ -134,7 +167,57 @@ export class ExecutionPlanComparisonPropertiesView extends ExecutionPlanProperti
 		if (this._model.secondaryElement?.properties) {
 			secondaryProps = this._model.secondaryElement.properties;
 		}
-		this.populateTable(columns, this.convertPropertiesToTableRows(primaryProps, secondaryProps));
+
+		const tableRows = this.convertPropertiesToTableRows(primaryProps, secondaryProps);
+		this.setSummaryElement(this.getExpensivePropertySummary(tableRows));
+		this.populateTable(columns, tableRows);
+	}
+
+	private getExpensivePropertySummary(tableRows: { [key: string]: string }[]): string[] {
+		let summary: string[] = [];
+
+		tableRows.forEach(row => {
+			const rowName = row.name['text'];
+			if (row.primary && row.secondary) {
+				const primaryText = row.primary['text'].split(' ');
+				const secondaryTitle = row.secondary['title'].split(' ');
+
+				if (primaryText.length === secondaryTitle.length && primaryText.length <= 2 && secondaryTitle.length <= 2) {
+					const MAX_PROPERTY_SUMMARY_LENGTH = 3;
```

Comment:
Why only 3? Is there going to be any indication that there's actually more - they just aren't displayed? (and any way for users to then see these other ones)?

Reply:
@Charles-Gagnon We have to have a limit - there could theoretically be 20 properties that are different.  The goal of this window was to help users quickly identify three or five (we picked 3 to start) important changes.  Determining what is "most important" is subjective and will likely cause opinions from users.

Reply:
I wonder if it'd be better to just add an info icon next to each entry then when hovered over will display the summary text. That way you get around the crowdedness of the current solution while also not trying to guess what ones are the "most" important. That's also a feature we use in a lot of other tables which display information like this and could let you customize the summary to include even more useful information (for example - links to docs on the properties and what they mean)

Reply:
This icon could also just be the current "difference" icon being used - but be something that when moused over included this extra information.

Reply:
(if you do something like this I would also suggest allowing the provider to give a "summary" for each difference. And then just have a simplistic default option like you do currently if none is provided. This would let providers give domain-specific summaries for each specific property as needed while not making it a pain to support new unsupported properties)